experimental/dds/tree2/src/test/feature-libraries/schemaIndex.spec.ts

```diff
@@ -64,6 +70,17 @@ describe("SchemaIndex", () => {
 		}
 	});
 
+	describe("codec", () => {
+		makeEncodingTestSuite(
+			makeCodecFamily([[0, codec]]),
+			testCases as EncodingTestData<SchemaData, Format>,
```

Comment:
If you declare the type of `testCases` to be `EncodingTestData<SchemaData, Format>` rather than `EncodingTestData<unknown, unknown>`, you won't need this cast here.